src/types/skin/SkinImage.php

```diff
@@ -32,6 +32,8 @@ public function __construct(
 
 	public static function fromLegacy(string $data) : SkinImage{
 		switch(strlen($data)){
+			case 0:
```

Comment:
heh?

Reply:
1.12 doesn't have image height and width and players don't have to have capes, but I also see the problem if players used skins like this

Reply:
Fixed